packages/api-generator/src/helpers/excludes.js

```diff
@@ -59,6 +59,30 @@ module.exports = {
       'open-delay',
       'return-value',
     ],
+    'v-otp-input': [
+      'appendIcon',
+      'backgroundColor',
+      'color',
+      'rules',
+      'dense',
+      'error',
+      'errorCount',
+      'errorMessages',
+      'height',
+      'hideDetails',
+      'hint',
+      'label',
+      'light',
+      'loaderHeight',
+      'loading',
+      'messages',
+      'persistentHint',
+      'prependIcon',
+      'readonly',
+      'success',
+      'successMessages',
+      'validateOnBlur',
```

Comment:
All the validation stuff still seems to work, it just doesn't show messages. Should proper validation be added? If you're not actually using any of the v-input functionality why extend it?

Reply:
Added `v-input` as the originated field..
Actually there is not a lot of inheritance here from `v-input`.
Just the computed `classes()` binding & the mouse events triggering. (onMouseDown / onMouseUp).

@KaelWD do you recommend on removing it?

Adding an error / success message(s) (Post processing) is a good idea on my perspective - but I wish to make sure the efforts are on track of getting approved.. ;)